packages/mint-components/src/components/sqm-portal-container/sqm-portal-container.tsx

```diff
@@ -33,9 +32,40 @@ export class PortalContainer {
    * @uiEnum ["none", "xxx-small", "xx-small", "x-small", "small", "medium", "large", "x-large", "xx-large", "xxx-large", "xxxx-large"]
    */
   @Prop() gap: Spacing = "xxx-large";
+
   /** @uiName Minimum width */
   @Prop() minWidth?: string;
 
+  /**
+   * @uiName Display
+   * @uiType string
+   * @uiEnum ["grid", "flex"]
+   */
+  @Prop() display: "grid" | "flex" = "grid";
+
+  /**
+   * @uiName Justify content
+   * @uiType string
+   * @uiEnum ["start", "center", "end", "space-between", "space-around", "space-evenly"]
+   */
+  @Prop() justifyContent?:
+    | "start"
+    | "center"
+    | "end"
+    | "space-between"
+    | "space-around"
+    | "space-evenly";
+
+  /**
+   * @uiName Maximum width
+   */
+  @Prop() maxWidth?: string = "100%";
+
+  /**
+   * @uiName Center content
+   */
+  @Prop() center: boolean;
```

Comment:
Given your types in the view, this should be optional

Reply:
This prop is now obsolete with `justifyContent` (since that has an option for `center`) and thus has been removed.